sonic-net/sonic-mgmt
# [dhcp_server][ansible] Enable dhcp_server in mx when deploy-mg

### Description of PR


Summary:
Fixes # (issue)

### Type of change





### Back port request

### Approach
#### What is the motivation for this PR?
Enable dhcp_server for mx when deploy-mg

#### How did you do it?
1. Use `config load_minigraph --override_config` to load minigraph.
2. Generate mx specified golden_config_db

#### How did you verify/test it?
1. Deploy-mg in mx device, configuration is as expected.
2. Deploy-mg in non-mx device, configuration is as espected.

#### Any platform specific information?

#### Supported testbed topology if it's a new test case?

### Documentation
